src/python/serenity_types/pricing/derivatives/options/valuation.py

```diff
@@ -230,7 +244,7 @@ class OptionValuationResult(CamelModel):
     Delta X qty X contract_size, the delta exposure expressed in qty of underlying.
     """
 
-    delta_ccy: float
+    delta_ccy: Optional[float]
```

Comment:
according to code, this is also optional

Reply:
Yes.